Examples/ObjectiveC/LightExample.m

```diff
@@ -0,0 +1,70 @@
+//
+//  LightExample.m
+//  Examples
+//
+//  Created by Jordan Kiley on 6/22/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+#import "LightExample.h"
+@import Mapbox;
+
+@interface LightExample () <MGLMapViewDelegate>
+
+@end
+
+NSString *const MBXExampleLight = @"LightExample";
+
+@implementation LightExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Set the map style to Mapbox Light Style version 9. The map's source will be queried later in this example.
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
```

Comment:
Though, using the Light style in an example that deals with a completely different type of “light” could be confusing.